### DISCUSSION TITLE
Getting an error when running vector store

### DISCUSSION THREAD

**Discussion by alekoii** [langflow-ai/langflow#341] [Q&A]
Hi, 
I'm getting the following error when running vector store 
"Error building node chroma: The API type provided is invalid. Please select one of the api types 'azure' azure_ad" 'open_ai'"

Where do I add the api type? 

Thanks

**Comment by rodrigosnader** [comment]
Hi @alekois, I hope this is solved in the new version. Don't hesitate to reach out on our [discord](http://discord.gg/EqksyE2EX9) channel!

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
